src/transformers/models/rt_detr/configuration_rt_detr.py

```diff
@@ -55,6 +55,8 @@ class RTDetrConfig(PretrainedConfig):
         use_timm_backbone (`bool`, *optional*, defaults to `False`):
             Whether to load `backbone` from the timm library. If `False`, the backbone is loaded from the transformers
             library.
+        freeze_backbone_batch_norm (`bool`, *optional*, defaults to `True`):
+            Whether to freeze the batch normalization layers in the backbone.
```

Comment:
Name in docs should match with parameter name
```suggestion
        freeze_backbone_batch_norms (`bool`, *optional*, defaults to `True`):
            Whether to freeze the batch normalization layers in the backbone.
```